client/src/utils/superblock-map-titles.ts

```diff
@@ -0,0 +1,25 @@
+import i18next from 'i18next';
+import { SuperBlocks } from '../../../config/certification-settings';
+
+// these are keys from i18n translations.json files
+enum SuperBlockI18nKeys {
+  Certification = 'learn.cert-map-estimates.certs',
+  Challenges = 'learn.cert-map-estimates.challenges'
```

Comment:
It's not used, but I left it in. What I had planned was to remove the cert projects from the legacy RWD and rename the button to say "Legacy RWD Challenges" instead of "Legacy RWD Certification". But I couldn't remove the cert projects since many of the i18n's are using the legacy RWD as their first superblock. So I think this will get used at some point. With that, do you still think we should remove this? I don't mind.

Reply:
It doesn't seem great to leave code in in anticipation of future use.  I'd be happy to create an issue to track this, though.